Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM debian:latest
+
+# Install packages
+RUN apt update && apt install -y \
+    curl \
+    gnupg \
+    wget
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+ && apt-get install -y nodejs
+
+# Clean up run image
+RUN apt remove -y \
+    curl \
+ && apt-get purge -y --auto-remove \
+                  -o APT::AutoRemove::RecommendsImportant=false \
+ && rm -rf /var/lib/apt/lists/* \
+           /tmp/*
+
+# Install app
+WORKDIR /root/app

```

Comment:
According to the docker docs this sets the working directory for the following commands so you don't need to use absolute paths for the copy operations

Reply:
https://docs.docker.com/engine/reference/builder/#workdir if you're curious

Reply:
Yeah, you're right.  Will fix.